docs/Project-Configuration/Controlling-when-tasks-run.md

```diff
@@ -0,0 +1,63 @@
+# Controlling when tasks run
+
+There are three ways to control the scheduling of builds/tasks on a project's waterfall page.
+
+In short:\
+**Cron:** activates builds/tasks on existing mainline commits based on a specified schedule.\
+**BatchTime:** sets an interval of time in minutes that Evergreen should wait before activating builds/tasks.\
```

Comment:
nit: we're pretty inconsistent about this in the codebase, but in the UI we use "Batch Time" so i think we should also use that in the docs (or we can just use "batchtime" which ise used elsewhere in the documentation). Also if we're capitalizing both words here, we should do that for periodic builds as well.